mesonbuild/cmake/interpreter.py

```diff
@@ -828,17 +828,17 @@ def configure(self, extra_cmake_options: T.List[str]) -> CMakeExecutor:
         self.fileapi.setup_request()
 
         # Run CMake
-        mlog.log()
+        mlog.debug()
         with mlog.nested():
-            mlog.log('Configuring the build directory with', mlog.bold('CMake'), 'version', mlog.cyan(cmake_exe.version()))
-            mlog.log(mlog.bold('Running CMake with:'), ' '.join(cmake_args))
-            mlog.log(mlog.bold('  - build directory:         '), self.build_dir.as_posix())
-            mlog.log(mlog.bold('  - source directory:        '), self.src_dir.as_posix())
-            mlog.log(mlog.bold('  - toolchain file:          '), toolchain_file.as_posix())
-            mlog.log(mlog.bold('  - preload file:            '), preload_file.as_posix())
-            mlog.log(mlog.bold('  - trace args:              '), ' '.join(trace_args))
-            mlog.log(mlog.bold('  - disabled policy warnings:'), '[{}]'.format(', '.join(disable_policy_warnings)))
-            mlog.log()
+            mlog.debug('Configuring the build directory with', mlog.bold('CMake'), 'version', mlog.cyan(cmake_exe.version()))
+            mlog.debug(mlog.bold('Running CMake with:'), ' '.join(cmake_args))
```

Comment:
Hmm, I think I'd probably keep the first line at least as a regular `mlog.log()`.

Reply:
I removed this as the following line will always come right before it:
```
Executing subproject <proj_name> method cmake 
```
And this line is likely to also be logged right before:
```
<proj_name>| Found CMake: /usr/bin/cmake (3.22.2)
```